CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# FSAStyles Changelog
+
+## Version 1.0.0-epimorphics
```

Comment:
I think we suggested holding this at `0.1.0-epimorphics` for now, until we've had a chance to discuss numbering with Anna